WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -176,15 +176,15 @@ - (NSDictionary *)fb_tree
 
 - (NSDictionary *)fb_tree:(nullable NSSet<NSString *> *)excludedAttributes
 {
-  id<FBXCElementSnapshot> snapshot = [self fb_takeSnapshot:YES];
+  id<FBXCElementSnapshot> snapshot = [self fb_standardSnapshot];
   return [self.class dictionaryForElement:snapshot
                                 recursive:YES
                        excludedAttributes:excludedAttributes];
 }
 
 - (NSDictionary *)fb_accessibilityTree
 {
-  id<FBXCElementSnapshot> snapshot = [self fb_takeSnapshot:YES];
+  id<FBXCElementSnapshot> snapshot = [self fb_standardSnapshot];
```

Comment:
Custom or intentional change?

Reply:
this should be standard because the source element type is always of XCUIApplication type